kernel/kernel-impl/src/main/java/org/sakaiproject/authz/impl/SakaiSecurity.java

```diff
@@ -479,6 +504,20 @@ String makeCacheKey(String userId, String function, String reference, boolean is
                 }
             }
         }
+	if (checkSwapped) {
```

Comment:
lookups for /content/group/NNNN is stored in the content cache.
lookups for /realm/* are generalized back to /site